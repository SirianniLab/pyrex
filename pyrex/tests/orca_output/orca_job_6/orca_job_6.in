!rhf sto-3g 


*xyz 0 1
C    0.029069777995    0.010788943258   -0.000016256628
O   -1.127677677201   -0.239742268804   -0.030154107424
O    1.099927291999    0.508548150937   -0.022917651901
H    0.414741506056   -2.253263997302    0.456727702729
H   -0.320451621399   -2.141342518830    0.385753195877
*


