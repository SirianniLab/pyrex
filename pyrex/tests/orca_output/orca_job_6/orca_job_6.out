
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #          Department of theory and spectroscopy      #
                  #               Directorship: Frank Neese             #
                  #        Max Planck Institute fuer Kohlenforschung    #
                  #                Kaiser Wilhelm Platz 1               #
                  #                 D-45470 Muelheim/Ruhr               #
                  #                      Germany                        #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 4.2.0 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Suceptibility
   Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
   Alexander A. Auer      : GIAO ZORA, VPT2
   Ute Becker             : Parallelization
   Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLED
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Miquel Garcia          : C-PCM Hessian, Gaussian charge scheme
   Yang Guo               : DLPNO-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Benjamin Helmich-Paris : CASSCF linear response (MC-RPA)
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : AUTO-CI
   Lucas Lang             : DCDCAS
   Dagmar Lenk            : GEPOL surface, SMD
   Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization
   Dimitrios Manganas     : Further ROCIS development; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Original ROCIS implementation
   Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density
   Barbara Sandhoefer     : DKH picture change effects
   Avijit Sen             : IP-ROCIS
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Bernardo de Souza      : ESD, SOC TD-DFT
   Georgi Stoychev        : AutoAux, RI-MP2 NMR
   Willem Van den Heuvel  : Paramagnetic NMR
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse,             : VdW corrections, initial TS optimization,
                  C. Bannwarth                     DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Andreas Klamt, Michael Diedenhofen            : otool_cosmo (COSMO solvation model)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model
   Lars Goerigk                                  : TD-DFT with DH, B97 family of functionals
   V. Asgeirsson, H. Jonsson                     : NEB implementation
   FAccTs GmbH                                   : IRC, NEB, NEB-TS, Multilevel, MM, QM/MM, CI optimization
   S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 4.2.3
 For citations please refer to: https://tddft.org/programs/libxc/

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


----- Orbital basis set information -----
Your calculation utilizes the basis: STO-3G
   H-Ne       : W. J. Hehre, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2657 (1969).
   Na-Ar      : W. J. Hehre, R. Ditchfield, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2769 (1970).
   K,Ca,Ga-Kr : W. J. Pietro, B. A. Levy, W. J. Hehre and R. F. Stewart, J. Am. Chem. Soc. 19, 2225 (1980).
   Sc-Zn,Y-Cd : W. J. Pietro and W. J. Hehre, J. Comp. Chem. 4, 241 (1983).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================


INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.in
|  1> !rhf sto-3g 
|  2> 
|  3> 
|  4> *xyz 0 1
|  5> C    0.029069777995    0.010788943258   -0.000016256628
|  6> O   -1.127677677201   -0.239742268804   -0.030154107424
|  7> O    1.099927291999    0.508548150937   -0.022917651901
|  8> H    0.414741506056   -2.253263997302    0.456727702729
|  9> H   -0.320451621399   -2.141342518830    0.385753195877
| 10> *
| 11> 
| 12> 
| 13> 
| 14>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  C      0.029070    0.010789   -0.000016
  O     -1.127678   -0.239742   -0.030154
  O      1.099927    0.508548   -0.022918
  H      0.414742   -2.253264    0.456728
  H     -0.320452   -2.141343    0.385753

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 C     6.0000    0    12.011    0.054934    0.020388   -0.000031
   1 O     8.0000    0    15.999   -2.131002   -0.453047   -0.056983
   2 O     8.0000    0    15.999    2.078561    0.961017   -0.043308
   3 H     1.0000    0     1.008    0.783748   -4.258052    0.863090
   4 H     1.0000    0     1.008   -0.605566   -4.046551    0.728968

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     1.183950528261     0.00000000     0.00000000
 O      1   2   0     1.181111560367   167.03435943     0.00000000
 H      1   2   3     2.341643321237    87.78138865   180.04653235
 H      4   1   2     0.747042657834    71.00562086     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     2.237342254525     0.00000000     0.00000000
 O      1   2   0     2.231977382703   167.03435943     0.00000000
 H      1   2   3     4.425064580463    87.78138865   180.04653235
 H      4   1   2     1.411706033662    71.00562086     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 3 groups of distinct atoms

 Group   1 Type C   : 6s3p contracted to 2s1p pattern {33/3}
 Group   2 Type O   : 6s3p contracted to 2s1p pattern {33/3}
 Group   3 Type H   : 3s contracted to 1s pattern {3}

Atom   0C    basis set group =>   1
Atom   1O    basis set group =>   2
Atom   2O    basis set group =>   2
Atom   3H    basis set group =>   3
Atom   4H    basis set group =>   3

Checking for AutoStart:
The File: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.gbw exists
Trying to determine its content:
     ... Fine, the file contains calculation information
     ... Fine, the calculation information was read
     ... Fine, the file contains a basis set
     ... Fine, the basis set was read
     ... Fine, the file contains a geometry
     ... Fine, the geometry was read
     ... Fine, the file contains a set of orbitals
     ... Fine, the orbitals can be read
     => possible old guess file was deleted
     => GBW file was renamed to GES file
     => GES file is set as startup file
     => Guess is set to MORead
     ... now leaving AutoStart

------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------

                         BASIS SET STATISTICS AND STARTUP INFO

 # of primitive gaussian shells          ...   33
 # of primitive gaussian functions       ...   51
 # of contracted shells                  ...   11
 # of contracted basis functions         ...   17
 Highest angular momentum                ...    1
 Maximum contraction depth               ...    3
 Integral package used                   ... LIBINT
 Integral threshhold            Thresh   ...  1.000e-10
 Primitive cut-off              TCut     ...  1.000e-11


------------------------------ INTEGRAL EVALUATION ----------------------------


 * One electron integrals 
 Pre-screening matrix                    ... done
 Shell pair data                         ... done (   0.000 sec)

-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   24
 Basis Dimension        Dim             ....   17
 Nuclear Repulsion      ENuc            ....     67.3802133740 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... LIBINT
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 1.931e-01
Time for diagonalization                   ...    0.005 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.004 sec
Total time needed                          ...    0.009 sec

---------------------
INITIAL GUESS: MOREAD
---------------------
Guess MOs are being read from file: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.ges
Input Geometry matches current geometry (good)
Input basis set matches current basis set (good)
MOs were renormalized
MOs were reorthogonalized (Cholesky)
                      ------------------
                      INITIAL GUESS DONE (   0.0 sec)
                      ------------------
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
                      *** Initiating the SOSCF procedure ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  0   -186.16910430 -186.1691042979  0.000012  0.000012  0.000018  0.000002
               *** Restarting incremental Fock matrix formation ***
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   1 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :         -186.16910430 Eh           -5065.91887 eV

Components:
Nuclear Repulsion  :           67.38021337 Eh            1833.50882 eV
Electronic Energy  :         -253.54931767 Eh           -6899.42769 eV
One Electron Energy:         -389.75373634 Eh          -10605.73835 eV
Two Electron Energy:          136.20441867 Eh            3706.31066 eV

Virial components:
Potential Energy   :         -370.62172629 Eh          -10085.12989 eV
Kinetic Energy     :          184.45262200 Eh            5019.21102 eV
Virial Ratio       :            2.00930582


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -5.5127e-10  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    1.7829e-05  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    2.2310e-06  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    8.0670e-06  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    1.1491e-05  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.scfp) ****
             **** ENERGY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.en.tmp) ****
             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.scfp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -20.360749      -554.0441 
   1   2.0000     -20.351687      -553.7975 
   2   2.0000     -11.265169      -306.5408 
   3   2.0000      -1.439436       -39.1691 
   4   2.0000      -1.370814       -37.3017 
   5   2.0000      -0.682558       -18.5733 
   6   2.0000      -0.657169       -17.8825 
   7   2.0000      -0.624927       -17.0051 
   8   2.0000      -0.584902       -15.9160 
   9   2.0000      -0.557920       -15.1818 
  10   2.0000      -0.388578       -10.5737 
  11   2.0000      -0.388541       -10.5727 
  12   0.0000       0.297023         8.0824 
  13   0.0000       0.313437         8.5291 
  14   0.0000       0.575598        15.6628 
  15   0.0000       0.657590        17.8939 
  16   0.0000       1.302074        35.4312 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 C :    0.455139
   1 O :   -0.230665
   2 O :   -0.222288
   3 H :   -0.017606
   4 H :    0.015420
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 C s       :     2.995297  s :     2.995297
      pz      :     0.859267  p :     2.549564
      px      :     0.846574
      py      :     0.843723
  1 O s       :     3.860981  s :     3.860981
      pz      :     1.565240  p :     4.369684
      px      :     1.215099
      py      :     1.589345
  2 O s       :     3.860260  s :     3.860260
      pz      :     1.573716  p :     4.362028
      px      :     1.332716
      py      :     1.455596
  3 H s       :     1.017606  s :     1.017606
  4 H s       :     0.984580  s :     0.984580


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 C :    0.330839
   1 O :   -0.170651
   2 O :   -0.158871
   3 H :   -0.015566
   4 H :    0.014250

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 C s       :     3.007585  s :     3.007585
      pz      :     0.866169  p :     2.661576
      px      :     0.933175
      py      :     0.862232
  1 O s       :     3.737320  s :     3.737320
      pz      :     1.562098  p :     4.433332
      px      :     1.282726
      py      :     1.588508
  2 O s       :     3.734814  s :     3.734814
      pz      :     1.570263  p :     4.424057
      px      :     1.385190
      py      :     1.468604
  3 H s       :     1.015566  s :     1.015566
  4 H s       :     0.985750  s :     0.985750


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 C      5.5449     6.0000     0.4551     3.9296     3.9296     0.0000
  1 O      8.2307     8.0000    -0.2307     2.3421     2.3421    -0.0000
  2 O      8.2223     8.0000    -0.2223     2.3570     2.3570    -0.0000
  3 H      1.0176     1.0000    -0.0176     0.9997     0.9997     0.0000
  4 H      0.9846     1.0000     0.0154     0.9998     0.9998     0.0000

  Mayer bond orders larger than 0.100000
B(  0-C ,  1-O ) :   1.9501 B(  0-C ,  2-O ) :   1.9740 B(  1-O ,  2-O ) :   0.3791 
B(  3-H ,  4-H ) :   0.9886 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.931 sec
Sum of individual times     ....       0.254 sec  ( 27.3%)

Fock matrix formation       ....       0.203 sec  ( 21.8%)
Diagonalization             ....       0.000 sec  (  0.0%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.035 sec  (  3.8%)
Initial guess               ....       0.003 sec  (  0.3%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.0%)
SOSCF solution              ....       0.013 sec  (  1.4%)

-------------------------   --------------------
FINAL SINGLE POINT ENERGY      -186.169104298403
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.gbw
Electron density file                           ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_6/orca_job_6.scfp
The origin for moment calculation is the CENTER OF MASS  = ( 0.000009,  0.000018 -0.000003)

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:     -0.04831       4.01925      -0.76786
Nuclear contribution   :      0.08804      -4.11896       0.78961
                        -----------------------------------------
Total Dipole Moment    :      0.03973      -0.09971       0.02175
                        -----------------------------------------
Magnitude (a.u.)       :      0.10952
Magnitude (Debye)      :      0.27838



--------------------
Rotational spectrum 
--------------------
 
Rotational constants in cm-1:     1.747141     0.370323     0.305557 
Rotational constants in MHz : 52377.979262 11101.989993  9160.366831 

 Dipole components along the rotational axes: 
x,y,z [a.u.] :    -0.002700    -0.109486     0.000109 
x,y,z [Debye]:    -0.006864    -0.278292     0.000276 

 

Timings for individual modules:

Sum of individual times         ...        2.055 sec (=   0.034 min)
GTO integral calculation        ...        0.934 sec (=   0.016 min)  45.5 %
SCF iterations                  ...        1.121 sec (=   0.019 min)  54.5 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 2 seconds 745 msec
